packages/react-charting/src/components/HorizontalBarChart/HorizontalBarChart.base.tsx

```diff
@@ -67,77 +67,73 @@ export class HorizontalBarChartBase extends React.Component<IHorizontalBarChartP
     let datapoint: number | undefined = 0;
     return (
       <div className={this._classNames.root} onMouseLeave={this._handleChartMouseLeave}>
-        {data!.map((points: IChartProps, index: number) => {
-          if (points.chartData && points.chartData![0] && points.chartData![0].horizontalBarChartdata!.x) {
-            datapoint = points.chartData![0].horizontalBarChartdata!.x;
-          } else {
-            datapoint = 0;
-          }
-          points.chartData![1] = {
-            legend: '',
-            horizontalBarChartdata: {
-              x: points.chartData![0].horizontalBarChartdata!.y - datapoint!,
-              y: points.chartData![0].horizontalBarChartdata!.y,
-            },
-            color: palette.neutralLight,
-          };
+        <FocusZone direction={FocusZoneDirection.vertical} handleTabKey={FocusZoneTabbableElements.all}>
+          {data!.map((points: IChartProps, index: number) => {
```

Comment:
Can you help me understand. What all elements in focusZone will be focusable. The ones marked as data-is-focusable = true?

Reply:
And tabbing within a focus zone will move to the next focus zone. Is this correct?

Reply:
> Can you help me understand. What all elements in focusZone will be focusable. The ones marked as data-is-focusable = true?

yes

> And tabbing within a focus zone will move to the next focus zone. Is this correct?

yes, tabbing within a focus zone will move to the next focusable element

BTW, I had to undo these changes because tabbing also moved focus to the stacked bars. Previously, the focus only moved to the stacked bars when the left/right arrow key was pressed.

Reply:
Ok. Interested to know what you learned and what were the blockers adopting a single FocusZone element.